sormas-ui/src/main/java/de/symeda/sormas/ui/utils/GridExportStreamResource.java

```diff
@@ -129,6 +131,10 @@ public InputStream getStream() {
 
 				Set<?> selectedRows = getSelectedRows != null ? getSelectedRows.get() : Collections.emptySet();
 
+				if (isNull(selectedRows)) {
+					selectedRows = Collections.emptySet();
+				}
+
 				if (selectedRows.isEmpty()) {
```

Comment:
Would be cleaner to use CollectionUtils.isEmpty(selectedRows)

Reply:
In this case if (CollectionUtils.isEmpty(selectedRows)) {
					selectedRows = Collections.emptySet();
				}
it will always comes in. And it was not what was expected.
However, I will adjust the changes.